### PR TITLE
[SIL-51] Add basic dependecies

### PR DESCRIPTION
**Pull request by Krzysiudan**
https://silentus-au.atlassian.net/jira/software/projects/SIL/boards/1?selectedIssue=SIL-51

Changes: 
- added DI Hilt dependency 
- added coroutines
- added retrofit
- added ktlint
- updated other dependencies.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -47,17 +50,39 @@ android {
...
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
-    implementation 'androidx.activity:activity-compose:1.3.1'
+    //Compose
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.compose.material3:material3:1.0.0-alpha11'
+    implementation 'androidx.compose.material3:material3:1.1.0-alpha06'
+
+    //DI - Hilt
+    implementation "com.google.dagger:hilt-android:2.45"
```
Comment (kezc) [inline app/build.gradle:74]: nitpick: if we extract version of compose maybe we could extract version of hilt and coroutines too

File: app/src/main/java/com/appunite/loudius/LoudiusApplication.kt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.appunite.loudius
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class LoudiusApplication : Application() {
```
Comment (kezc) [inline app/src/main/java/com/appunite/loudius/LoudiusApplication.kt]: nitpick: unnecessary parentheses

**Review by kezc — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (Krzysiudan) [reply app/build.gradle:74]: Hmm, why not.. 
Then we agree to extract every dependency version? Maybe we could use a gradle kotlin DSL?

**Review by nowakweronika — APPROVED** [review]
lgtm

**Pull request merged**